website/src/components/auth/BeginnersGuide.tsx

```diff
@@ -124,7 +124,7 @@ export const BeginnersGuide: React.FC<React.HTMLAttributes<HTMLDivElement>> = (
             Decentraland on-chain assets.
         </p>
           <p>
-            Access the Marketplace at{" "}
```

Comment:
@2fd I think we need to use it when a tag is on the next line, also we should apply Prettier